static/src/javascripts/projects/commercial/modules/hosted/video.ts

```diff
@@ -1,17 +1,16 @@
 import { initHostedYoutube } from 'commercial/modules/hosted/youtube';
 
 export const initHostedVideo = (): Promise<void> => {
-	const youtubeIframe: NodeListOf<HTMLElement> = document.querySelectorAll(
+	const playerContainers = document.querySelectorAll<HTMLElement>(
```

Comment:
Technically, these aren’t `iframe`s yet! They’re just `div`s that will be turned into `iframe`s after they’ve been passed to `setupPlayer`.